Admin/Admin.php

```diff
@@ -2946,6 +2946,16 @@ public function checkAccess($action, $object = null)
             }
         }
 
+        return $access;
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
currently, until we uncomment the code from the interface, there is no inheritedDoc.

Can you please add the PHPDoc here?

And can you create a separate PR for the next Major after we merged this one where you set inheritDoc back and uncomment the interface methods?